packages/tui/internal/components/list/list.go

```diff
@@ -61,6 +63,14 @@ var simpleListKeys = listKeyMap{
 		key.WithKeys("j"),
 		key.WithHelp("j", "next list item"),
 	),
+	CtrlP: key.NewBinding(
+		key.WithKeys("ctrl+p"),
+		key.WithHelp("ctrl+p", "previous list item"),
+	),
+	CtrlN: key.NewBinding(
+		key.WithKeys("ctrl+n"),
+		key.WithHelp("ctrl+n", "next list item"),
+	),
```

Comment:
let's just add `ctrl+p` to the `Up` binding, and `ctrl+n` to the `Down` binding, won't need to handle these separately. the alpha keys are handled separately only bc for some lists we shouldn't allow this (like when you need to still be able to type into a filter/etc). you then should be able to leave the `Update` switch statement untouched.

Reply:
Good call! Done!